elements/pfe-tabs/BaseTabs.ts

```diff
@@ -74,6 +83,8 @@ export abstract class BaseTabs extends LitElement {
 
   id: string = this.id || getRandomId(this.localName);
 
+  @property({ reflect: true }) inset?: InsetVariant;
```

Comment:
Do we need this property to be reflected to the attribute?

Reply:
After discussion in chat, you are correct.